stan/math/prim/scal/prob/cauchy_log.hpp

```diff
@@ -1,22 +1,7 @@
 #ifndef STAN_MATH_PRIM_SCAL_PROB_CAUCHY_LOG_HPP
 #define STAN_MATH_PRIM_SCAL_PROB_CAUCHY_LOG_HPP
 
-#include <stan/math/prim/scal/meta/is_constant_struct.hpp>
-#include <stan/math/prim/scal/meta/partials_return_type.hpp>
```

Comment:
Don't we want to include what we explicitly use in the classes even if it's included indirectly through another header.  Your call.  I'm going to just approve this and let you sort out what you want.